documentation/decisions/adr/000N-wiki-platform.md

```diff
@@ -0,0 +1,349 @@
+- **Wiki.js** if we want to prioritize data access and open source tools but are willing to compromise on maintenance costs and usability
+
+## Comparison Matrix
+
+- ✅ Feature available, meets requirement
+- ❌ Feature not available, does not meet requirement
+- 🔄 Partial feature, limited feature availability, feature in progress or undergoing improvements
+- 1-3 Strength level
+- ❓Unknown
+
+| Factor                      | Confluence | Notion | GitHub Wiki | GitBook | Wiki.js |
+| --------------------------- | :--------: | :----: | :---------: | :-----: | :-----: |
+| Usability                   |     3      |   2    |      1      |    2    |    1    |
+| Public Access               |    🔄     |  🔄   |     ✅      |   ✅    |   ✅    |
+| Content Review              |    🔄     |   ❌   |     ❌      |   ✅    |   🔄   |
+| Version History             |     ✅     |  🔄   |     ✅      |   ✅    |   🔄   |
+| Multi-Media                 |     ✅     |   ✅   |     ❌      |   ✅    |   ✅    |
+| I18n                        |    🔄     |  🔄   |     ❌      |   ✅    |   ✅    |
+| Web Analytics               |     ✅     |   ✅   |     ❌      |   ✅    |   🔄   |
+| Onboarding Cost Efficiency  |     2      |   2    |      3      |    2    |    1    |
+| Maintenance Cost Efficiency |     2      |   2    |      3      |    2    |    1    |
+| Authority to Operate        |     ✅     |   ❌   |     ✅      |   ❌    |   ✅    |
+| External Contributions      |     ✅     |   ✅   |     🔄     |   🔄   |   🔄   |
+| Data Access                 |    🔄     |  🔄   |     ✅      |   ✅    |   ✅    |
+| Machine Readability         |    🔄     |   ✅   |     ❌      |   🔄   |   ✅    |
+| Open Source                 |     ❌     |   ❌   |     ❌      |   ❌    |   ✅    |
+
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+### Confluence
+
+[Confluence](confluence) is a Software as a Service (SaaS) documentation and collaboration tool offered by Atlassian that organizes content into "spaces" and offers a series of templates and components that can be used to create custom documentation for internal and external stakeholders.
+
+#### Details
+
+- **Hosting:** SaaS
+- **Pricing:** [$5.75 (standard) or $11 (premium) per user per month](confluence-pricing)
+- **Public Access:** Supported, but limited to individual pages or entire spaces
+- **Content Review:** Partial support, limited to drafts and not enforceable
+- **Version History:** Supported by default
+- **Supported Media**
+  - Markdown style text
+  - File uploads
+  - Image embedding
+  - Video embedding
+  - Diagrams
+- **I18n:** Limited third party plugins for automating translation
+- **Web Analytics:** [Google Analytics plugin](confluence-ga) available, also native analytics with premium tier
+- **Open Source Status:** Propietary
+- **External Contributions:** Only supported in public spaces (without review)
+- **Data Access:** Limited access via API
+
+#### Pros
```

Comment:
One HUGE pro for Confluence — and I'm not suggesting it be chosen — is that its commenting system facilitates conversation within the wiki without updating content. The feature is so fantastic it might warrant adding to the comparison matrix.

Reply:
I added comments to the list of decision criteria. Fortunately GitBook also supports commenting